assets/js/data/store-notices/index.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { createReduxStore, register } from '@wordpress/data';
+import { controls } from '@wordpress/data-controls';
```

Comment:
What are these for and why do we need them?

Reply:
I don't actually know. This was in the other stores.